sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2.scala

```diff
@@ -67,6 +68,46 @@ object HiveThriftServer2 extends Logging {
     }
   }
 
+  def runScript(sqlContext: HiveContext) {
+    val hiveConf = sqlContext.hiveconf
+    val script = hiveConf.get("hive.server2.init.script", null)
```

Comment:
Actually Spark SQL can load the init scripts, and the default value of `hive.server2.global.init.file.location` is $HIVE_CONF_DIR, not $SPARK_CONF_DIR or the current classpath. 

So this PR makes the same thing which is already supported, probably what we want is changing the default value to $SPARK_CONF_DIR?